src/Listbox.js

```diff
@@ -190,20 +201,25 @@ export const ListboxOption = {
     this.context.unregisterOptionRef(this.value)
   },
   render(h) {
+    const isMultilistbox = this.context.props.multiple
```

Comment:
What are your thoughts on checking like `const isMultiple = Array.isArray(this.context.props.value)` (that way we don't require an additional prop here?)

Reply:
As we already pass the Listbox props to context via `provide`, I think using the `multiple` prop makes it clearer what we aim for.

If you think about getting rid of the `multiple` prop on the Listbox itself, I would not recommend this. Magically using multi select behavior because an empty array was passed as the initial value doesn't feel right.

Reply:
> magically using multi select behavior because an empty array was passed as the initial value doesn't feel right.

I agree with you on this part. I don't think passing an empty array would be valid.

I guess I'm thinking about this from a futuristic point of view. Personally I'm developing a similar package to this for React (https://github.com/mitchbne/react-listbox), so I'd love to hear your thoughts on this. Futuristically, I want this entire Listbox to be able to do a few things:

1) Select single input (typeahead)
2) Select multiple inputs (typeahead)
3) Select single input (input search where ListboxItems filter)
4) Select multiple inputs (input search where ListboxItems filter)

I don't think there's a scope for more ideas (according to W3 docs) but I don't want to be naive - the more 'future proof' the better.

So keeping in mind this idea of 'multi select' as well as 'filter select', what's better?

Should we pass in more props:  `mutli={boolean}` and `filterable={boolean}` (we could run into an issue where we need to just keep adding props here).

Or should we deterministically handle in the Listbox component whether or not it detects a `ListboxButton`, or `ListboxFilter`...I guess the problem is, what should it do in the scenario that you create both of these as children to the `Listbox`?

Reply:
My best bet is to *not* use an additional prop for filterable listboxes, because it is more of an addition that could be used alongside the typeahead behavior. Even so I can't imagine a case where one would like to have a ListboxButton with typeahead together with a ListboxFilter, it would be possible to do that.

In contrast multi select cannot be used alongside single select behavior. It is either one or the other. So here a prop makes more sense to me.

Reply:
> In contrast multi-select cannot be used alongside single-select behaviour.

I think that statement stands out to me the most.

If it can be used alongside existing behaviour --> favour deterministic handling.
If it cannot be used alongside existing behaviour --> favour a second prop.

By the way, I'm happy to close this comment with the conclusion that we'll use `multi` as a prop.